[2025-05-30T13:18:29.302+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_ecobici.procesar_datos manual__2025-05-30T13:18:17.221327+00:00 [queued]>
[2025-05-30T13:18:29.322+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_ecobici.procesar_datos manual__2025-05-30T13:18:17.221327+00:00 [queued]>
[2025-05-30T13:18:29.323+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-05-30T13:18:29.363+0000] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): procesar_datos> on 2025-05-30 13:18:17.221327+00:00
[2025-05-30T13:18:29.367+0000] {standard_task_runner.py:60} INFO - Started process 273 to run task
[2025-05-30T13:18:29.370+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_ecobici', 'procesar_datos', 'manual__2025-05-30T13:18:17.221327+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/etl_ecobici.py', '--cfg-path', '/tmp/tmp61f9qluo']
[2025-05-30T13:18:29.372+0000] {standard_task_runner.py:88} INFO - Job 25: Subtask procesar_datos
[2025-05-30T13:18:29.475+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_ecobici.procesar_datos manual__2025-05-30T13:18:17.221327+00:00 [running]> on host 21812af8c39c
[2025-05-30T13:18:29.672+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_ecobici' AIRFLOW_CTX_TASK_ID='procesar_datos' AIRFLOW_CTX_EXECUTION_DATE='2025-05-30T13:18:17.221327+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-30T13:18:17.221327+00:00'
[2025-05-30T13:18:29.687+0000] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2025-05-30T13:18:29.688+0000] {spark_submit.py:341} INFO - Spark-Submit cmd: spark-submit --master spark:7077 --conf spark.master=spark://spark:7077 --name arrow-spark --verbose --queue default /app/dags/scripts/procesar_ecobici.py
[2025-05-30T13:18:29.731+0000] {spark_submit.py:492} INFO - JAVA_HOME is not set
[2025-05-30T13:18:29.741+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 423, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark:7077 --conf spark.master=spark://spark:7077 --name arrow-spark --verbose --queue default /app/dags/scripts/procesar_ecobici.py. Error code is: 1.
[2025-05-30T13:18:29.766+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=etl_ecobici, task_id=procesar_datos, execution_date=20250530T131817, start_date=20250530T131829, end_date=20250530T131829
[2025-05-30T13:18:29.825+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 25 for task procesar_datos (Cannot execute: spark-submit --master spark:7077 --conf spark.master=spark://spark:7077 --name arrow-spark --verbose --queue default /app/dags/scripts/procesar_ecobici.py. Error code is: 1.; 273)
[2025-05-30T13:18:29.866+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-05-30T13:18:29.916+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
